openpype/hosts/maya/plugins/publish/extract_look.py

```diff
@@ -186,10 +204,11 @@ def process(self,
 
         self.log.debug(" ".join(subprocess_args))
         try:
-            run_subprocess(subprocess_args)
+            output = run_subprocess(subprocess_args, logger=self.log)
         except Exception:
             self.log.error("Texture .rstexbin conversion failed",
                            exc_info=True)
+            self.log.debug(output)
```

Comment:
```suggestion
```
You can't log the `output` var if the try clause failed since the var wouldn't be set.